justfile

```diff
@@ -314,8 +312,18 @@ ln-dlc-node-test args="": docker
 
 # Runs background Docker services
 docker:
+    #!/usr/bin/env bash
+    set -euxo pipefail
     docker-compose up -d
 
+    # wait a few seconds to ensure that Docker containers started
+    sleep 2
```

Comment:
This asks for flakyness :/ We might need some busy wait here.

Reply:
we actually don't have to wait. bitcoin is up instantly.